.github/workflows/scorecard.yml

```diff
@@ -3,16 +3,17 @@ name: Scorecard supply-chain security
 permissions: read-all
 
 on:
-  # For Branch-Protection check. Only the default branch is supported. See
-  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
-  branch_protection_rule: # To guarantee Maintained check is occasionally updated. See
+  branch_protection_rule:
+    # To guarantee Maintained check is occasionally updated. See
+    # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
```

Comment:
Err -- please tell me that this is not a cronjob aimed at creating (just) the impression of "maintained"-ness, @planetf1 : If it were, I'd violently demand this to be dropped: If no-one maintains the project this fact should become visible. Otherwise this whole scorecard thing really is just a hoax.

Reply:
If that is indeed what this action is doing, then agreed -- having an artificial commit to meet a metric seems like a bad idea.

Reply:
This code **isn't performing a commit**, it's just [triggering if the branch protection rules are updated](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#branch_protection_rule) so that the scorecard can be refreshed. 

The comment isn't the best placed and has likely moved a little after merges - it's a generic comment (or could be removed).

However the comment is accurate, in that the [check it refers to](https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained) i is looking at recent commits and using that to determine an overall score. There is no particular threshold. .. and yes it can be gamed by introducing the noise of a unnecessary commit.

I agree the whole area of these 'heuristics' in general is debated a lot (see the [chaoss](https://chaoss.community/) community!)

Some organizations may use these metrics to understand more about a project. Overall we score fairly highly...

liboqs is in fact being scanned by openssf directly - the results are [here](https://securityscorecards.dev/viewer/?uri=github.com/open-quantum-safe/liboqs)

The action allows a few more checks to work, enables badging, raises events (security) for tracking - and in fact is needed for smaller projects they've yet to scan.